src/webfaf/templates/problems/list.html

```diff
@@ -61,8 +61,26 @@
         </form>
       </div>
       <div id="problem-list">
-          {{ problems_table(problems) }}
-     </div>
+        <table class="table table-striped table-bordered table-condensed">
+          <tr>
```

Comment:
Can you put this into the `thead` tag?

It will make all tables have consistent structure/styling. - https://github.com/abrt/faf/commit/bf7bf30c2b155227491f171a03df8fc7ddf992f3